dist-material/release-docs/NOTICE

```diff
@@ -905,4 +905,31 @@ be obtained at:
   * HOMEPAGE:
     * https://github.com/catapult-project/catapult
 
+------
+
+===========================================================================
```

Comment:
As MIT license you don't need to update NOTICE. NOTICE is for Apache 2.0 only. Please remove this part.